projects/Canary/Controllers/MessagesController.cs

```diff
@@ -16,6 +16,55 @@ namespace canary.Controllers
     [ApiController]
     public class MessagesController : ControllerBase
     {
+        /// <summary>
+        /// Inspects FSH message
+        /// </summary>
+        /// <returns></returns>
+        [HttpPost("Messages/sushi/Inspect")]
+        public async Task<(string results, List<Dictionary<string, string>> issues)> NewFshPost()
+        {
+            string input = await new StreamReader(Request.Body, Encoding.UTF8).ReadToEndAsync();
+
+            if (!String.IsNullOrEmpty(input))
+            {
+                //Verify input is FSH, check for Alias?
+                if (input.Trim().StartsWith("Alias:")) // FSH?
```

Comment:
Is there something in the FSH specification that we can use to check for FSH-formatted messages?